avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java

```diff
@@ -89,14 +89,14 @@ public void configure(@NonNull Map<String, ?> configs, boolean isKey) {
 
         UUID schemaVersionIdFromRegistry = null;
         if (this.schemaVersionId == null) {
-            log.info("Schema Version Id is null. Trying to register the schema.");
+            log.debug("Schema Version Id is null. Trying to register the schema.");
```

Comment:
Are we logging elsewhere that schema is getting registered?

Reply:
Yes